src/components/data-management/ProjectDetails.jsx

```diff
@@ -35,7 +38,7 @@ const ProjectDetails = ({ width, height }) => {
   const activeProject = useSelector((state) => state.projects[activeProjectUuid]) || false;
```

Comment:
perhaps the property in `state.projects.meta.activeProject` could be `state.projects.meta.activeProject.activeProjectUuid`? Just to be more explicit.

If you look at these two lines it looks a little bit weird how one is getting activeProject (renamed to Uuid) to get activeProject in the next one (which is the actual activeProject).